cmd/kubeadm/app/preflight/checks.go

```diff
@@ -182,6 +184,38 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	return nil, errors
 }
 
+// FileContentCheck checks that the given file contains the string Content.
+type FileContentCheck struct {
+	Path    string
+	Content []byte
+}
+
+func (fcc FileContentCheck) Check() (warnings, errors []error) {
```

Comment:
the other functions don't do that, and since I am not changing any value in the struct, I didn't opt to do that.